lint/tests/test_lint.py

```diff
@@ -153,6 +153,58 @@ def test_lint_failing(capsys):
     assert err == ""
 
 
+def test_ref_existent_relative(capsys):
+    with _mock_lint("check_path") as mocked_check_path:
```

Comment:
Either looks fine to me; I don't think it makes much difference. Our tests for linting in general could be cleaned up, but that's a task for another time.